AppController/lib/app_dashboard.rb

```diff
@@ -48,6 +47,17 @@ module AppDashboard
   #   true if the AppDashboard was started successfully, and false otherwise.
   def self.start(login_ip, uaserver_ip, public_ip, private_ip,
       persistent_storage, secret)
+
+    # Pass the secret key and our public IP address (needed to connect to the
+    # AppController) to the app.
+    Djinn.log_run("cp #{APPSCALE_HOME}/AppDashboard/lib/secret_key.py /tmp/secret_key.py")
+    Djinn.log_run("cp #{APPSCALE_HOME}/AppDashboard/lib/local_host.py /tmp/local_host.py")
+    Djinn.log_run("cp #{APPSCALE_HOME}/AppDashboard/lib/uaserver_host.py /tmp/uaserver_host.py")
```

Comment:
Actually we modified that variable a while ago and that's why the dashboard broke.
Please refer to https://github.com/AppScale/appscale/commit/c4170768ff1b84744edfa909210a20d15437a064#diff-3728529fdd148dba907df39fdadd6521L53